sunpy/util/parfive_helpers.py

```diff
@@ -12,16 +13,16 @@
 
 # Overload the parfive downloader class to set the User-Agent string
 class Downloader(parfive.Downloader):
-
     @wraps(parfive.Downloader.__init__)
     def __init__(self, *args, **kwargs):
         headers = kwargs.pop("headers", None)
-
         if headers is None or 'User-Agent' not in headers:
             headers = {'User-Agent':
                        f"sunpy/{sunpy.__version__} parfive/{parfive.__version__} "
                        f"aiohttp/{aiohttp.__version__} python/{sys.version[:5]}"}
-
         kwargs["headers"] = headers
-
+        # Works with conftest to hide the progress bar.
+        progress = os.environ.get("HIDE_PARFIVE_PROGESS", None)
+        if progress:
+            kwargs["progress"] = False
```

Comment:
We should upstream this.

Reply:
We should, I was looking at the test logs and saw the progress bar and wanted to turn it off in bulk. This was my quick patch.